src/workerd/api/node/tests/http-server-nodejs-test.js

```diff
@@ -1111,6 +1114,33 @@ export const testScheduled = {
   },
 };
 
+export const testConfigurableHighWaterMark = {
+  async test(_ctrl, env) {
+    {
+      await using server = http.createServer({ highWaterMark: 9999 });
+      strictEqual(server.highWaterMark, 9999);
+    }
+
+    {
+      // Node.js supports 1.1 as a value for highWaterMark
+      await using server = http.createServer({ highWaterMark: 1.11 });
```

Comment:
nit: should this read "1.11" instead ?

Reply:
`highWaterMark` is expected to be a number.